app.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const { getAllTopics, getAllEndPoints } = require("./controllers/topics.controller");
+const { customErrors, psqlErrors, serverErrors } = require('./errors/errors.js');
+const app = express();
+
+app.get("/api/topics", getAllTopics);
+app.get('/api', getAllEndPoints)
+
+app.use(customErrors);
+app.use(psqlErrors);
+app.use(serverErrors);
```

Comment:
Are all of these error handlers being used yet? If not, they should be removed until they are.